README.md

```diff
@@ -27,7 +27,10 @@ terminal (Mac OS), `oh-my-zsh` with `powerlevel9k` theme and `powerline nerd-fon
 
 1. Install Ruby (prefably, version > 2.1)
 2. Install the patched fonts of powerline nerd-font and/or font-awesome.
-3. Clone this repository with `git clone https://github.com/athityakumar/colorls.git`
+3. Clone this repository to `~/bin` (create if not exist or clone elsewhere) with
+```sh
+cd ~/bin && git clone https://github.com/athityakumar/colorls.git
+```
 4. Navigate to this cloned directory : `cd colorls`
```

Comment:
This works because you already are in the  `~/bin` folder and you stay there even after the git clone.
You could add the `cd colorls` to the initial command as in `cd ~/bin && git clone https://github.com/athityakumar/colorls.git && cd colorls `.